static/js/components/dashboard/courses/GradeDetailPopup.js

```diff
@@ -64,36 +94,51 @@ const labelBestGrade = (runs: Array<CourseRun>): Array<[CourseRun, boolean]> =>
   ]));
 };
 
+const labelBestExamGrade = (exams: Array<ProctoredExamResult>): Array<[ProctoredExamResult, boolean]> => {
+  let bestGrade = exams.reduce((acc, exam) => (
+    exam.percentage_grade > acc ? exam.percentage_grade : acc
+  ), 0);
+
+  return exams.map(exam => ([exam, exam.passed && exam.percentage_grade === bestGrade]));
+};
+
 const renderRunRows = R.compose(
   R.addIndex(R.map)(renderRunRow),
-  labelBestGrade
+  labelBestEdxGrade
+);
+
+const renderExamRows = R.compose(
+  R.addIndex(R.map)(renderExamRow),
+  labelBestExamGrade
 );
 
-const dialogTitle = (course: Course): string => (
-  `${course.title} - completed edX Course runs`
+const dialogTitle = (course: Course, gradeType: GradeType): string => (
+  `${course.title} - ${gradeType === EDX_GRADE ? "completed edX Course runs" : "Completed Exams"}`
```

Comment:
Should `Completed Exams` be `completed Exams` here, for consistency with the other string?